src/utils/søknadsbehandling/vilkår/vilkårUtils.ts

```diff
@@ -74,27 +81,51 @@ const getBehandlingsinformasjonStatus = <VilkårKey extends keyof Behandlingsinf
     }
 };
 
+const mapToVilkårsinformasjonUføre = (uføre: GrunnlagsdataOgVilkårsvurderinger['uføre']): Vilkårsinformasjon[] => [
+    {
+        status:
+            uføre === null
+                ? VilkårVurderingStatus.IkkeVurdert
+                : uføre?.resultat === UføreResultat.HarUføresakTilBehandling
+                ? VilkårVurderingStatus.Uavklart
+                : uføre?.resultat === UføreResultat.VilkårOppfylt
+                ? VilkårVurderingStatus.Ok
+                : VilkårVurderingStatus.IkkeOk,
+        vilkårtype: Vilkårtype.Uførhet,
+        erStartet: uføre !== null,
+    },
+];
+
+const mapToVilkårsinformasjonAlder = (alder: GrunnlagsdataOgVilkårsvurderinger['alder']): Vilkårsinformasjon[] => [
+    {
+        status:
+            alder === null
+                ? VilkårVurderingStatus.IkkeVurdert
+                : alder?.resultat === Aldersresultat.HarAlderssakTilBehandling
+                ? VilkårVurderingStatus.Uavklart
+                : alder?.resultat === Aldersresultat.VilkårOppfylt
+                ? VilkårVurderingStatus.Ok
+                : VilkårVurderingStatus.IkkeOk,
+        vilkårtype: Vilkårtype.Alderspensjon,
+        erStartet: alder !== null,
+    },
+];
+
```

Comment:
Tenker att dette er greit for now, men kanskje vi skal på sikt prøve å generalisere håndteringen av uførevilkår og aldersvilkår? Har inget konkret forslag nå, men bara en tanke. 😊

Reply:
Ja, enig! Er ganske mye her vi kan prøve å samle litt mer 😊 Men tar det senere tenker jeg! Når vi kan wrappe opp litt mer